src/java/org/apache/cassandra/locator/AbstractReplicationStrategy.java

```diff
@@ -351,6 +351,11 @@ protected void validateReplicationFactor(String rf) throws ConfigurationExceptio
     {
         try
         {
+            if (Integer.parseInt(rf) < DatabaseDescriptor.getMinimumKeyspaceRF())
```

Comment:
nit: perhaps dry this up since we call parseInt twice now:

```
Integer parsedRf = Integer.parseInt(rf);
if (parsedRF < DatabaseDescriptor.getMinimumKeyspaceRF())
```